src/hamster-cli.py

```diff
@@ -277,8 +276,9 @@ def export(self, *args):
         end_time = end_time or start_time.replace(hour=23, minute=59, second=59)
         facts = self.storage.get_facts(start_time, end_time)
 
-        writer = reports.simple(facts, start_time.date(), end_time.date(), export_format)
-
+        writer = reports.simple(
```

Comment:
Ugh! That's a deterioration IMO

Reply:
This is just because the line is > 88 characters, so it is wrapped. Might mean the line length should be made a bit longer, then.

Reply:
I just looked at the current code to see if there's a line length that would minimize the diff, but it seems the current code is not all that consistent, so that probably won't really work.